hack/update-bazel.sh

```diff
@@ -20,6 +20,15 @@ set -o pipefail
 export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+if LANG=C sed --help 2>&1 | grep -q GNU; then
```

Comment:
note to self: this snippet is in so many scripts.. should probably be ripped into hack/lib/util.sh as a function.

Reply:
Indeed, found a couple of them.